ada/testsuite/tests/dsl_unparse/test.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import, division, print_function
+
+import os
+import subprocess
+
+
+tmpdir = os.path.abspath(os.curdir)
+unparse_dest = os.path.join(
+    os.path.abspath(os.curdir),
+    'lal.lkt'
+)
```

Comment:
I believe you can replace all the above with just: `unparse_dest = os.path.abspath('lal.lkt')`